bin/test_travis.sh

```diff
@@ -99,7 +99,7 @@ ctest --output-on-failure
 # Python
 if [[ "${WITH_PYTHON}" == "yes" ]] && [[ "${WITH_SAGE}" != "yes" ]]; then
     cd symengine/python
-    nosetests -v
+    py.test -v
```

Comment:
I guess there might be, but nosetests ignore markers such as `@pytest.mark.xfail` etc. I will try to see if there is a way to support both. Any particular reason to prefer nosetests?